templates/atlus_hbn.bt

```diff
@@ -0,0 +1,62 @@
+typedef ubyte u8;
+typedef short s16;
+typedef ushort u16;
+typedef uint u32;
+typedef float f32;
+
+typedef struct
+{
+	u8 entrynum <name = "Entry Number">;
+	u8 Unknown1;
+	u8 Unknown2;
+	u8 Unknown3;
+	byte Unknown4;
+	u8 Unknown5;
+	u8 Unknown6;
+	u8 Unknown7;
+	float Unknown8 <name = "Y Pos">;
+	float Unknown9 <name = "Z Pos">;
+	float Unknown10 <name = "X Pos">;
+	float Unknown11;
+	float Unknown12;
+	float Unknown13;
+} THBNEntries1 <name = "Type 1 Entries">;
+
+typedef struct
+{
+	u8 entrynum <name = "Entry Number">;
+	u8 Unknown1;
+	u8 Unknown2;
+	u8 Unknown3;
+	float Unknown4 <name = "X Pos">;
+	float Unknown5 <name = "Z Pos">;
+	float Unknown6 <name = "Y Pos">;
+	float rot <name = "Rotation">;
+} THBNEntries2 <name = "Type 2 Entries">;
+
+typedef struct TableSegment
```

Comment:
This is not a good name for this struct as it's not represenative of its contents.

Reply:
Name should be FbnFile or something equivalent that represents the contents better